docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -0,0 +1,197 @@
+create visual art for a specific topic.
+
+| image:images/tag_cloud.png[Tag cloud]
+
+|===
+
+[float]
+[[create-aggregation-based-panel]]
+=== Create an aggregation-based panel
+
+Choose the type of panel you want to create, then use the editor to configure the options. Each panel type supports different options. 
+
+. From the dashboard, click *Create new*, then click *Aggregation based* on the *New visualization* window.
+
+.. Click the type of panel you want to create.
+
+.. Click the data source you want to visualize.
+
+. From the editor, add the <<aggregation-reference,aggregations>> you want to visualize, then click *Update*. 
++
+NOTE: To use *auto* from the *Minimal interval* dropdown, the *Date Histogram* aggregation must be the primary field. 
+
+. To change the order, drag the aggregations along the editor. 
++ 
+[role="screenshot"]
+image:images/bar-chart-tutorial-3.png[Option to change the order of aggregations]
+
+. To customize the series colors, click the series in the legend, then select the color you want to use.
++ 
+[role="screenshot"]
+image:images/aggregation-based-color-picker.png[Color picker]
+
+[float]
+[[try-it-aggregation-based-panel]]
+=== Try it: Create an aggregation-based bar chart
+
+You collected data from your web server, and you want to visualize and analyze the data on a dashboard. To create a dashboard panel of the data, create
+a bar chart that displays the top five log traffic sources for every three hours.
+
+[float]
+==== Add the data and create the dashboard
+
+Add the sample web logs data that you'll use to create the bar chart, then create the dashboard.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample web logs*, click *Add data*.
+
+. Open the main menu, then click *Dashboard*.
+
+. Click *Create dashboard*.
+
+[float]
+==== Open and set up the aggregation-based bar chart
+
+Open the bar chart visualization builder and change the time range. 
+
+. On the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click *Aggregation based > Vertical bar*.
+
+.. On the *Choose a source* window, click *kibana_sample_data_logs*.
+
+. Change the <<set-time-filter, date time filter>>> to *Last 7 days*.
+
+[float]
+[[tutorial-configure-the-bar-chart]]
+==== Create the bar chart
+
+To create the bar chart, add a <<bucket-aggregations,bucket aggregation>>, then add the terms sub-aggregation to display the top values.
+
+. Add a *Buckets* aggregation.
+
+.. Click *Add*, then select *X-axis*.
+
+.. From the *Aggregation* dropdown, select *Date Histogram*.
+
+.. Click *Update*.
++
+[role="screenshot"]
+image:images/bar-chart-tutorial-1.png[Bar chart with sample logs data]
+
+. To show the top five log traffic sources, add a sub-bucket aggregation.
+
+.. Click *Add*, then select *Split series*.
++
+TIP: Aggregation-based panels support a maximum of three *Split series*.
```

Comment:
How about moving this tip to https://www.elastic.co/guide/en/kibana/7.10/aggregation-reference.html? It is kind of buried on this page and seems out of place. I would expect this type of information on a reference documentation page like "Aggregation reference" and less on a tutorial documentation page like this page.

Reply:
This is now in the list of benefits to using aggregation-based visualizations.